openmeetings-core/src/main/java/org/apache/openmeetings/core/remote/KRoom.java

```diff
@@ -118,21 +123,6 @@ public void onStopBroadcast(KStream stream, final StreamProcessor processor) {
 		//FIXME TODO permission can be removed, some listener might be required
 	}
 
-	public void leave(final StreamProcessor processor, final Client c) {
```

Comment:
We don't need this method anymore. It would lead to circular reference otherwise.